src/core/advanced_mode.h

```diff
@@ -178,6 +178,7 @@ namespace librealsense
         void get_color_white_balance(white_balance_control* ptr) const;
         void get_color_auto_white_balance(auto_white_balance_control* ptr) const;
         void get_color_power_line_frequency(power_line_frequency_control* ptr) const;
+        void get_sub_preset(hdr::hdr_control* ptr) const;
```

Comment:
As mentioned elsewhere, don't use the term "sub_preset". You can "get_hdr_preset" or "get_hdr_control", etc...